CSharp/Library/Microsoft.Bot.Builder/Dialogs/PromptDialog.cs

```diff
@@ -172,24 +172,24 @@ protected string DefaultTooManyAttempts
         /// <param name="prompt"> The prompt.</param>
         /// <param name="retry"> What to display on retry.</param>
         /// <param name="tooManyAttempts"> What to display when user didn't say a valid response after <see cref="Attempts"/>.</param>
-        /// <param name="options"> The prompt choice values.</param>
+        /// <param name="values"> The prompt choice values.</param>
         /// <param name="attempts"> Maximum number of attempts.</param>
         /// <param name="promptStyler"> The prompt styler.</param>
-        /// <param name="descriptions">Descriptions for each prompt.</param>
+        /// <param name="titles">Descriptions for each prompt.</param>
         /// <param name="speak"> The Speak tag (SSML markup for text to speech).</param>
         /// <param name="retrySpeak"> What to display on retry Speak (SSML markup for text to speech).</param>
         /// <param name="recognizer"> Entity Recognizer to parse the message content.</param>
-        public PromptOptions(string prompt, string retry = null, string tooManyAttempts = null, IReadOnlyList<T> options = null, int attempts = 3, PromptStyler promptStyler = null, IReadOnlyList<string> descriptions = null, string speak = null, string retrySpeak = null, IPromptRecognizer recognizer = null)
+        public PromptOptions(string prompt, IReadOnlyList<string> titles = null, IReadOnlyList<T> values = null, string retry = null, string tooManyAttempts = null, int attempts = 3, PromptStyler promptStyler = null, string speak = null, string retrySpeak = null, IPromptRecognizer recognizer = null)

```

Comment:
This is a breaking change and probably better not to reorder the parameters in the constructor.